LICENSE

```diff
@@ -2,8 +2,8 @@
  * ----------------------------------------------------------------------------
  * "THE DERIVED BEER-WARE LICENSE" (Revision 1):
  * You can do whatever you want with this stuff. When you like it, just buy
- * Volker Buzek (@vobu) a beer or buy Peter Muessig (@pmuessig) a coke when
- * you see one of them.
+ * Volker Buzek (@vobu), Marco Beier (@wridgeu) or Matthias Oßwald (@matthiaso) 
```

Comment:
Actually I think Matthias is written with double s - https://people.sap.com/matthias.osswald

Reply:
@matz3's GitHub says Oßwald 😉

SAP replicates employee names everywhere. No chance to get non-ASCII chars in there (at least not 10 years ago) 🙈